src/cmd/build.js

```diff
@@ -4,6 +4,7 @@ import minimatch from 'minimatch';
 import {createWriteStream} from 'fs';
 import streamToPromise from 'stream-to-promise';
 import {fs} from 'mz';
+import parseJson from 'parse-json';
```

Comment:
Nit: I know that `parseJson` is how this import is named in the examples from 'parse-json` readme file, but I would prefer it named as `parseJSON`.